pages/common/conda-list.md

```diff
@@ -0,0 +1,24 @@
+# conda list
+
+> List installed packages in a conda environment.
+> More information: <https://docs.conda.io/projects/conda/en/stable/commands/list.html>.
+
+- List all packages in the current environment:
+
+`conda list`
+
+- List packages in the named environment:
+
+`conda list -n <environment>`
```

Comment:
```suggestion
`conda list {{[-n|--name]}} <environment>`
```
Either use long options by themselves or use option placeholders
Fix the rest of the page